Functions/Import/xASL_imp_CheckImportSettings.m

```diff
@@ -70,9 +70,8 @@
 	fprintf('%s\n', [num2str(nLeftBrackets) ' tokens detected in folderHierarchy']);
 	fprintf('%s\n', [num2str(nTokens) ' tokens detected in tokenOrdering']);
 
-	if nLeftBrackets~=nTokens
-		fprintf('Note that the () symbols are reserved to define tokens in folderHierarchy!');
-		error('Number of tokens of folderHierarchy and tokenOrdering should be the same');
+	if nLeftBrackets<nTokens
+		error('Number of tokens in folderHierarchy defined by () should be equal or higher than the length of tokenOrdering vector');
```

Comment:
I see your point, but I want to see the examples to ensure that we cannot reinforce () == number of tokens. Because this would make it much easier and clearer for the user.

Also, you can practice the use of articles, which article is missing in your error message? ;)

Reply:
Including those articles missing in what you wrote or excluding?